bins/client/src/ecs/systems/client_network.rs

```diff
@@ -39,6 +39,7 @@ use gv_game::{
 use crate::ecs::resources::{
     LastAcknowledgedUpdate, ServerCommand, UiNetworkCommand, UiNetworkCommandResource,
 };
+use gv_core::ecs::resources::net::MultiplayerRoomPlayer;
```

Comment:
Move this import to the group above, please, and run `cargo fmt`

Reply:
I should probably document it somewhere.. :) I try to keep to the following:
- external libraries
- std
- gv_*
- crate

Reply:
Good point, done. `cargo fmt` isn't giving me problems, since I'm using Rust nightly on Windows. Let me know if you know how to fix that.